UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs

```diff
@@ -226,7 +226,7 @@ public HungerState CalculateHungerState()
 		/// </summary>
 		private List<Sickness> immunedSickness;
 
-		public PlayerScript playerScript;
+		public PlayerScript PlayerScriptOwner;
```

Comment:
Why did you rename this back to PlayerScriptOwner, PlayerScript is fine?

Reply:
Like I said on Discord. camelCase on a public variable and in the context of that script it makes sense; because we're getting the owner of that healthMaster script